### PR TITLE
Sort "No Consultation Filed" patients to the top of Patient List page 

### PR DESCRIPTION
**Pull request by GokulramGHV**
## Proposed Changes

- sorted patients such that patients with no consultation are before patients with consultation

### Associated Issue

- coronasafe/care_fe#6635

## Merge Checklist

- [ ] Tests added/fixed
- [ ] Update docs in `/docs`
- [ ] Linting Complete
- [ ] Any other necessary step

_*Only PR's with test cases included and passing lint and test pipelines will be reviewed*_

@coronasafe/care-backend-maintainers @coronasafe/care-backend-admins

### CODE REVIEW

**Review by rithviknishad — COMMENTED** [review]
Would this approach work for consultations with readmissions?

**Comment by nihal467** [comment]
@GokulramGHV fix the issue ASAP

**Comment by Ashesh3** [comment]
Here is the conditions when we show the "No consultation filed" badge:
```
(!patient.last_consultation || patient.last_consultation?.facility !== patient.facility ||
(patient.last_consultation?.discharge_date && patient.is_active))
```

**Comment by GokulramGHV** [comment]
> Here is the conditions when we show the "No consultation filed" badge:
> 
> ```
> (!patient.last_consultation || patient.last_consultation?.facility !== patient.facility ||
> (patient.last_consultation?.discharge_date && patient.is_active))
> ```

@Ashesh3 Thanks!

**Comment by GokulramGHV** [comment]
@nihal467 Can you check  if it works for the readmissions? I've updated the sorting order.

**Comment by nihal467** [comment]
LGTM

**Review by rithviknishad — APPROVED** [review]

File: care/facility/api/viewsets/patient.py [hunk 1]
```diff
@@ -344,12 +344,31 @@ class PatientViewSet(
...
         # if filter_query:
         #     disease_status = filter_query if filter_query.isdigit() else DiseaseStatusEnum[filter_query].value
         #     return queryset.filter(disease_status=disease_status)
 
         # if self.action == "list":
         #     queryset = queryset.filter(is_active=self.request.GET.get("is_active", True))
+        queryset = queryset.order_by(
+            "-no_consultation_filed",
+            "modified_date",
+        )
```
Comment (sainak) [inline care/facility/api/viewsets/patient.py:371]: do this only if the action == list

**Review by sainak — APPROVED** [review]

**Review by Ashesh3 — APPROVED** [review]

**Review by sainak — APPROVED** [review]

**Review by Ashesh3 — APPROVED** [review]

**Comment by vigneshhari** [comment]
Resolve conflicts

**Review by vigneshhari — APPROVED** [review]

**Pull request merged**